frontend/javascripts/oxalis/model/bucket_data_handling/bucket.js

```diff
@@ -235,6 +236,16 @@ export class DataBucket {
     return this.state === BucketStateEnum.MISSING;
   }
 
+  isUnsynced(): boolean {
```

Comment:
good ideas and inspirations! I'll dwell on that a bit.

another idea: `isPartial` in the sense of "it's still missing something (i.e., the potentially existing data from the backend)"

Reply:
> If the backend is the onliest data source of "missing data" then I would prefer isMissingBackendData as it is the most straight forward imo. It does directly tell in what condition the bucket is in: It misses backend data.

Yes, this is true, but the caveat is that this can be confused with "isMissing" (i.e., the backend doesn't have any data for this bucket). This is why, I'm going for `needsBackendData` now. I hope this alright.